contact_us.qmd

```diff
@@ -0,0 +1,5 @@
+---
+  title: "Contact Us"
+---
+
+  To contact us, please send an email to **[to add contact email address]** 
```

Comment:
this would then be almost identical to the join us page - I wonder if either
1) we just crosslink to this page from the Join us page 
or
2) instead of email addresses, we link here to a (google) form where people can even leave anonymous feedback if they want

Reply:
@ywang-gilead I found https://support.google.com/a/users/answer/9308623?hl=en which explains how to embed a google form in a website. I would propose to
1) create a google form
2) get the embed code as html
3) save that as separate html file
4) include this after body in this doc, see https://quarto.org/docs/output-formats/html-basics.html#includes how to do that

then we have a nice clear separation of code (html vs. qmd) and the form is nicely integrated into the page